### PR TITLE
Update Minotaur from v1.2.1 to v2.2.0

### PR DESCRIPTION
**Pull request by triphora**

### CODE REVIEW

File: resources/gradle_scripts/fabric-publish.gradle [hunk 1]
```diff
@@ -180,31 +180,30 @@ tasks.getByName("curseforge").doLast {
     sendDiscordWebhook()
 }
 
-task modrinth(type: com.modrinth.minotaur.TaskModrinthUpload) {
-    onlyIf {
-        System.getenv().MODRINTH_TOKEN
-    }
-
-    token = System.getenv().MODRINTH_TOKEN
```
Comment (Hugman76) [inline resources/gradle_scripts/fabric-publish.gradle:188]: Where does the token go?

Comment (Hugman76) [reply resources/gradle_scripts/fabric-publish.gradle:188]: Oh, it's already the default one...

**Comment by Hugman76** [comment]
Maybe this could also support dependencies?
Like `modrinth_required_dependencies` & `modrinth_optional_dependencies`

**Comment by triphora** [comment]
done (remember to squash this PR)

**Comment by Hugman76** [comment]
Thanks a lot for the help! It is very appreciated

**Pull request merged**